### ISSUE TITLE
Unable to find version with pip install

### ISSUE THREAD

**Issue by Falc0nFight3r** [microsoft/onnxruntime#15389]
### Describe the issue

Whenever I try to run 'pip install onnxruntime-gpu', it doesn't ever find a version, even when copying it directly from the site

### To reproduce

I'm using python 3.9, and the latest pip install code, as it suggested me try. Other packages install just fine, including the normal version of onnxruntime, just that onnxruntime-gpu won't be found if I try and switch out for that.

### Platform

Windows

### OS Version

Windows 10 pro, version 22h2

### ONNX Runtime Installation

Other / Unknown

### ONNX Runtime Version or Commit ID

onnxruntime-gpu

### ONNX Runtime API

Python

### Architecture

X64

### Execution Provider

CUDA, Other / Unknown

### Execution Provider Library Version

CUDA 11.7

**Comment by snnn** [comment]
Please run your python.exe and execute the following code:
```
import platform
print(platform.architecture()[0])
```

**Comment by Falc0nFight3r** [comment]
It says 32 bit, and that's it

**Comment by snnn** [comment]
Yes. That's the reason. CUDA doesn't have Windows 32-bit support.
